timeless/views.py

```diff
@@ -178,6 +180,19 @@ def get_context_object_list_name(self):
         """
         return self.context_object_list_name
 
+    def sort_query(self, query):
+        ordering = request.args.get("ordering")
```

Comment:
@vladarefiev I'd name the paramenter ``orderBy`` instead of ``ordering``

Reply:
@amihaiemil ok, but `order_by` it's python style